modules/grafana/manifests/dashboards/rummager_index_size_dashboard.pp

```diff
@@ -0,0 +1,21 @@
+# == Class: grafana::dashboards::deployment_dashboard
+#
+# Automate the creation of app specific deployment Grafana dashboards
+#
+# === Parameters
+#
+# [*dashboard_directory*]
+#   The directory where the Grafana config json files are created
+#
+#
+class grafana::dashboards::rummager_index_size_dashboard (
+  $dashboard_directory = '/etc/grafana/dashboards',
+  $index_names = ['govuk', 'mainstream', 'government', 'detailed'],
+) {
+
+  file {
+    "${dashboard_directory}/rummager_index_size.json":
+    notify  => Service['grafana-server'],
+    content => template('grafana/dashboards/rummager_index_size.json.erb');
+  }
+}
```

Comment:
I don't think you should need this class, could you add it to this list? https://github.com/alphagov/govuk-puppet/blob/master/modules/grafana/manifests/dashboards.pp#L33-L41

Reply:
Good call, have simplified this now.